var/spack/repos/builtin/packages/libabigail/package.py

```diff
@@ -12,12 +12,14 @@ class Libabigail(AutotoolsPackage):
     homepage = "https://sourceware.org/libabigail"
     url      = "https://mirrors.kernel.org/sourceware/libabigail/libabigail-2.0.tar.gz"
     git      = "https://sourceware.org/git/libabigail.git"
+    maintainers = ['woodard']
 
     version('master', branch='master')
     version('2.0', sha256='3704ae97a56bf076ca08fb5dea6b21db998fbbf14c4f9de12824b78db53b6fda')
     version('1.8', sha256='1cbf260b894ccafc61b2673ba30c020c3f67dbba9dfa88dca3935dff661d665c')
 
     variant('docs', default=False, description='build documentation')
+    variant('rpm', default=False, description='build RPM tools')
```

Comment:
Would it make sense to have this variant only on linux::
```suggestion
    variant('rpm', default=False, when='platform=linux', description='build RPM tools')
```
?